.eslintrc.json

```diff
@@ -75,6 +74,7 @@
     "react/no-array-index-key": "off",
     "react/react-in-jsx-scope": "off",
     "react/require-default-props": "off",
+    "react/sort-comp": "off",
```

Comment:
This is all about sorting class members, which is largely irrelevant to our codebase. The errors it raised weren't worth worrying about.